src/scripts/tweaks.json

```
@@ -7,6 +7,7 @@
     "background_color": "#ecc84b"
   },
   "help": "https://github.com/AprilSylph/XKit-Rewritten/wiki/Features#tweaks",
+  "relatedTerms": [ "Blacklist" ],
```

"separat" already matches a preference, but since we're here, we might as well...

```suggestion
  "relatedTerms": [ "Blacklist", "Separator" ],
```

---

Hah, I had to go git blame the XK7 spring cleaning list to to figure out what this was referring to.